www/%username/widgets/index.html.spt

```diff
@@ -68,5 +68,25 @@ response.headers[b'Content-Security-Policy'] = csp
     <p><script src="giving.js"></script></p>
     % endif
 
+    <h3>{{ _("Shields.io Badges") }}</h3>
+
+    <p>{{ _("Use this code to add a badge to your website or GitHub:") }}</p>
```

Comment:
I see a couple of issues with this wording:

- "this code" is singular, but there are 4 different pieces of code
- "GitHub" is only one platform, I think "README" would be more generic and more precise at the same time (it says exactly where the badges are meant to be included)

Reply:
Addressed in https://github.com/liberapay/liberapay.com/pull/1111/commits/a3e34cef4e908b5f252d5d533fd3c562f0cdcd54. "Shields.io" is capitalized because that seems to be the way they write it.